linchpin/provision/roles/aws/tasks/provision_aws_sg.yml

```diff
@@ -8,4 +8,4 @@
     aws_secret_key: "{{ auth_var['aws_secret_access_key'] | default(omit) }}"
     rules: "{{ res_def['rules'] | aws_sg_rules('inbound') }}"
     rules_egress: "{{ res_def['rules'] | aws_sg_rules('outbound') }}"
-    no_log: "{{ not debug_mode }}"
+    vpc_id: "{{ res_def['vpc_id'] | default(omit) }}"
```

Comment:
@petr-balogh Could you please add no_log: "{{ not debug_mode }}" to the task indented to left. 
It would help us protecting credentials in case of failures.